config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'home#index'
+  resources :users
```

Comment:
If you are not using all the paths, you can only get what you need by doing something like:
```ruby
resources :users, only: [:index, :show, :create]
```
Source: [Rails docs](https://edgeguides.rubyonrails.org/routing.html#configuring-the-rails-router)

Reply:
Thanks, I will refactor asap